server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v70/DbVersion70.java

```diff
@@ -41,8 +41,11 @@ public void addSteps(MigrationStepRegistry registry) {
       .add(1911, "Add ORGANIZATIONS.DEFAULT_QUALITY_GATE_UUID", AddDefaultQualityGateUuidToOrganizations.class)
       .add(1912, "Create QUALITY_GATES.UUID", AddUuidToQualityGates.class)
       .add(1913, "Populate QUALITY_GATES.UUID", PopulateUuidOnQualityGates.class)
-      .add(1914, "Make QUALITY_GATES.UUID not nullable", MakeUuidNotNullableOnQualityGates.class)
```

Comment:
Why have you drop this migration ?

Reply:
Moreoever, it's no more consistent with H2 => https://github.com/SonarSource/sonarqube/blob/feature/eh/SONAR-10145/server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl#L232